manifest.json

```diff
@@ -27,5 +27,5 @@
         }
     ],
 
-    "manifest_version": 2
```

Comment:
(test message)
we need to support firefox as well

Reply:
(test message)
 at the moment chrome doesn't support manifest v2
we are working on other solutions.